x-pack/plugins/apm/public/context/environments_context/environments_context.tsx

```diff
@@ -39,7 +45,7 @@ export function EnvironmentsContextProvider({
 
   const { start, end } = useTimeRange({ rangeFrom, rangeTo, optional: true });
 
-  const { environments, status } = useEnvironmentsFetcher({
+  const { environments, status, environmentOptions } = useEnvironmentsFetcher({
```

Comment:
I realize this was in there before. But I'm not sure if `useEnvironmentsFetcher` should return `environmentOptions`. That seems like a responsibility of the component that displays the suggestions. Not a blocker though.